doc/usecases/core/UC_keyset_retrieve.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Retrieve `Key` from `KeySet`
```

Comment:
Again, I think this is only a scenario of lookup.

Reply:
No it's not. If you look carefully, the lookups return an index and this retrieves the key given a known index. This use case is also what makes iteration possible. No other use case says you should be able to access `Key`s by index.

In the current API the lookup is `ksSearch` and this is `ksAtCursor`.

Reply:
Then please call it `at` instead of `retrieve`.